inference-engine/include/ie_common.h

```diff
@@ -20,6 +20,8 @@
 #include <stdexcept>
 #include <iterator>
 
+#include <openvino/pp.hpp>
```

Comment:
`pp` is not a part of OpenVINO release package. I suppose we cannot use it in public headers.